documentation/SupportingDocuments/Population Density API - Algorithm Initial Proposal.md

```diff
@@ -0,0 +1,35 @@
+# Population Density Data API
+## Algorithm initial proposal
+
+This document serves as a reference model for the population density calculation algorithm. It provides an illustrative high-level overview of the steps proposed to determine population density. Designed for developers, this guide is created to enhance understanding and implementation of the algorithm. While it aims to enhance understanding and implementation, it is the responsibility of the developer to create their own algorithm to calculate population density.
+
+### Inputs for the algorithm:
+* UEs connection records data is received pseudonymized or hashed.
```

Comment:
Commonalties (https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md) guides to avoid using telco specific terms. "UE" is the very first example of term to avoid :)

Please replace it with "devices"

Reply:
solved